docs/components/morphotagger.rst

```diff
@@ -160,58 +160,123 @@ Command line:
 ----------------
 
 If you want to use our models from scratch, do the following
-(all the examples are for ru\_syntagrus corpus, change the filenames accordingly to invoke models for other languages):
+(all the examples are for ru\_syntagrus\_pymorphy model,
+change the filenames accordingly to invoke models for other languages):
 
 #. Download data
 
     .. code:: bash
 
-       python -m deeppavlov download morpho_ru_syntagrus_train
+       python -m deeppavlov download morpho_ru_syntagrus_pymorphy
 
    To perform all downloads in runtime you can also run all subsequent
    commands with ``-d`` key,
 
-#. To apply a pre-trained ru\_syntagrus model to ru\_syntagrus test
-   data, run
+#. To apply a pre-trained ru\_syntagrus\_pymorphy model to ru\_syntagrus test
+   data provided it was downloaded using the previous command, run
+
+   .. code:: bash
+
+     python -m deeppavlov.models.morpho_tagger morpho_ru_syntagrus_pymorphy \
+     > -f /.deeppavlov/downloads/UD2.0_source/ru_syntagrus/ru_syntagrus-ud-test.conllu
```

Comment:
```suggestion
     > -f ~/.deeppavlov/downloads/UD2.0_source/ru_syntagrus/ru_syntagrus-ud-test.conllu
```

Reply:
But it won't work because there is no `expand_path()` nor `expanduser()` in the path parsing...